### ISSUE TITLE
Adz support

### ISSUE THREAD

**Issue by toralux** [dirkwhoffmann/vAmiga#341]
It would have been great if vAmiga supported adz (zipped floppy format) in addition to adf as my huge game collection is mostly in adz format. The emulator is very promising. Thanks. :)

**Comment by dirkwhoffmann** [comment]
At the moment, I see two possibilities: 

1. To add a third party library such as minizip
2. To call gzip as a background task

I don't want to go for option 1, because vAmiga is 100% free of third-party code and I want to preserve this luxury. So there is still option 2. Interestingly, nobody seems to use this approach, so I am wondering if there is a reason 🤔.

**Comment by dirkwhoffmann** [comment]
Is there some site that provides ADZ files for download? I've never stumbled across them. I could produce my own ADZs by running gzip on an ADF, but I don't know if that's the right thing to do. I.e., it's unclear to me if an ADZ is a gzipped ADF or an ADF that has been run through a deflate encoder (i.e., a zip file without the zip header).

**Comment by mithrendal** [comment]
I also never saw one ... 😳 Is it supported by SAE ? If so then we could look into the implementation...

**Comment by toralux** [comment]
Adz is gzipped adf file: https://en.wikipedia.org/wiki/Amiga_Disk_File#ADZ

**Comment by dirkwhoffmann** [comment]
Supported in v0.9.9 which is now online.

**Comment by toralux** [comment]
It still don't work with my adz-files (tried on versions since 0.9.9, now on version 9.9.17.1). I will attach a demo that fails with error "The content of the file does not match its format".

Here is how I converted my adf-files to adz-files. These adz-files work in UAE.
`
gzip *.adf
ren *.adf.gz *.adz
`

[Crusaders_BackToBase.adf.gz](https://github.com/dirkwhoffmann/vAmiga/files/5880289/Crusaders_BackToBase.adf.gz)
(I had to rename it back from adz to adf.gz because github doesn't allow attatching files with adz-extension)
